django_fido/static/django_fido/js/u2f-registration.js

```diff
@@ -1,70 +1,71 @@
-U2F_FORM_ID = 'django-fido-u2f-form'
-U2F_ERROR_LIST_ID = 'django-fido-u2f-errors'
+const U2F_FORM_ID = 'django-fido-u2f-form'
+const U2F_ERROR_LIST_ID = 'django-fido-u2f-errors'
 // U2F request identifiers - shared between code and JS
-U2F_REGISTRATION_REQUEST = 'registration'
-U2F_AUTHENTICATION_REQUEST = 'authentication'
+const U2F_REGISTRATION_REQUEST = 'registration'
+const U2F_AUTHENTICATION_REQUEST = 'authentication'
 // Timeout for the U2F request
-U2F_TIMEOUT = 30
-
+const U2F_TIMEOUT = 30
+const U2F_ERROR_CODES = {
+    1: 'An error occurred while processing U2F request',
+    2: 'U2F request cannot be processed.',
+    3: 'Your configuration for U2F is not supported.',
+    4: 'The presented device is not eligible for this request.',
+    5: 'U2F request timed out.'
+}
 
 function addU2fError(message) {
-    var error_list = document.getElementById(U2F_ERROR_LIST_ID);
-    var new_item = document.createElement("li");
-    new_item.appendChild(document.createTextNode(message));
+    const error_list = document.getElementById(U2F_ERROR_LIST_ID);
+    const new_item = document.createElement("li");
+    new_item.classList.add('error');
+    new_item.appendChild(document.createTextNode(gettext(message)));
```

Comment:
`gettext` function should fallback to identity, if Django's jsi18n is not loaded.